content/en/docs/concepts/architecture/leases.md

```diff
@@ -78,3 +81,23 @@ spec:
 ```
 
 Expired leases from kube-apiservers that no longer exist are garbage collected by new kube-apiservers after 1 hour.
+
+You can disable API server identity leases by disabling the `APIServerIdentity`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+
+## Workloads {#custom-workload}
+
+Your own workload can define its own use of Leases. For example, you might run a custom
+{{< glossary_tooltip term_id="controller" text="controller" >}} where a primary or leader member
+performs steps that its peers do not. You define a Lease so that the controller replicas can select
```

Comment:
"steps" is more about a workflow, how about changing it to just "operations"?

```suggestion
performs operations that its peers do not. You define a Lease so that the controller replicas can select
```